SDK/MobileCenter/Microsoft.Azure.Mobile.UWP/Utils/DeviceInformationHelper.cs

```diff
@@ -15,22 +15,23 @@ namespace Microsoft.Azure.Mobile.Utils
     /// </summary>
     public class DeviceInformationHelper : AbstractDeviceInformationHelper
     {
-        private static string _cachedScreenSize = "unknown";
-        private static bool _didSetUpScreenSizeEvent;
-        private static readonly bool CanReadScreenSize;
         public static event EventHandler InformationInvalidated;
-        private static readonly object LockObject = new object();
         private static string _country;
-        private static readonly SemaphoreSlim DisplayInformationEventSemaphore = new SemaphoreSlim(0);
         private static readonly TimeSpan DisplayInformationTimeout = TimeSpan.FromSeconds(2);
```

Comment:
Timeout is used in case it takes too long for the screen size to be determined (a caller on in a background task needs to wait because it can only be determined from the main thread. see `IsAvailableAsync`)